### ISSUE TITLE
It would be nice for Holiday features to be client side decision or townhall option

### ISSUE THREAD

**Issue by Shadizar** [ldtteam/minecolonies-features#871]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my issue is not covered there.
- [x] I made sure that **this change request is not a duplicate of any existing change request.**

### Context 

- Minecraft Version: All supported
- MineColonies Version: All
- Related Mods and their Versions: N/A

## Is your feature request related to a problem?
Some people seem to really like holiday features, and others to really not...so, on a server, individual choice would be nice instead of a serverwide config
(Also because some ppl may celebrate one but not another, for example)

## Enhancement or Change description
For the Holiday features to either move to client config, or for it to be a townhall Options choice

### Notes or related things

---
## Viewers

* Add a thumbs-up to the change request to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by YumatanGames** [comment]
Yeah, I agree. I love Christmas, but many players are not Christians and their religion should be respected too. Maybe even adding some options for some religious holidays for other religions would be really a nice touch.

**Comment by Shadizar** [comment]
closing, as this was implemented...
